.circleci/config.yml

```diff
@@ -67,11 +67,26 @@ commands:
             - /home/circleci/.composer/cache/
       - store_artifacts:
           path: ./artifacts
-      - slack/status:
-          fail_only: true
-          failure_message: ':red_circle: M2: A $CIRCLE_JOB job has failed!'
-          only_for_branches: master
-          webhook: $SLACK_GREENKEEPER_WEBHOOK
+      - run:
+          name: "Setup branch string"
```

Comment:
Could we name something else?
You don't need to dismissing my 👍  this is minar thing

Reply:
Good point, I had originally named it that before all the changes. I'll update the name